src/content/docs/style-guide/processes-procedures/use-content-types-text-formats.mdx

```diff
@@ -60,7 +60,7 @@ Thr top of every doc begins with a set of metadata. Read on for more information
 
     <tr>
       <td>topics</td>
-      <td>Keywords related to your doc. Start each topic with a ** _ ** on a new line. 
+      <td>Keywords related to your doc. Start each topic with a `_`* on a new line. 
```

Comment:
We've renamed `topics` to `tags`. Should that be captured here?

Reply:
Yup! I've updated this doc.